integration/tests/cook/test_basic.py

```diff
@@ -209,6 +209,29 @@ def test_mea_culpa_retries(self):
         finally:
             util.kill_jobs(self.cook_url, [uuid])
 
+    def test_compute_cluster(self):
+        settings_dict = util.settings(self.cook_url)
+        # For now, we only have one compute cluster. This could be wrong with future
+        # refactors, but for now, this is the only compute cluster we have, so we should
+        # expect it.
+        expected_compute_cluster = settings_dict['mesos-compute-cluster-name']
+        expected_mesos_framework = settings_dict['mesos-framework-id']
+        job_uuid, resp = util.submit_job(self.cook_url)
+
+        try:
+            self.assertIsNotNone(expected_compute_cluster)
+            self.assertEqual(resp.status_code, 201, msg=resp.content)
+
+            instance = util.wait_for_instance(self.cook_url, job_uuid)
+            message = repr(instance)
+
+            self.assertIsNotNone(instance['compute-cluster'],message)
+            self.assertEqual(expected_compute_cluster,instance['compute-cluster']['compute-cluster-name'],message)
```

Comment:
I like the longer name. Because we don't have types in clojure, if you want to find something, the best way is to give a consistent---and unique--- name.

Reply:
We don't do that with other fields, e.g. `job-name`, `job-cpus`, `job-mem`. The convention is to not include the parent object type in the field names.

Reply:
We're not destructuring much on job name/cpus/mem. (And, I'd claim that not disambuguating name/cpus/mem for jobs may be a mistake.) We will be destructuring based on compute-cluster-name.... If you see a bare 'name' in a function, is it a job name, or a compute cluster name or ..... That's going to add additional friction in understanding code.

Reply:
I think it's clear from the name of the thing you're destructuring what the fields are.

It's also odd to do this for this one single map, when we haven't done it for any other maps.

Reply:
Ok. We're returning :name in the API response.